### ISSUE TITLE
Soldier NPCs

### ISSUE THREAD

**Issue by Bysokar** [StargateMC/StargateMCPublic#263]

**Comment by dodhod** [comment]
Don't we have wraith, goauld Jaffa and tauri?  I'm still ill by the way :c

**Comment by Bysokar** [comment]
Yep, but theyre not finished by a long shot and have issues.. ;)
Get well, then come back to us bud.
